Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            Console.WriteLine("Welcome to the self-confidence crusher! Also known as the BMI Calculator. Please input your information below.");
+            Console.WriteLine();
+
+            // First Person Start.
+            Console.WriteLine("Hello person one, please enter your details below.");
+
+            Person();
+
+            Console.WriteLine();
+
+            // Second Person Start
+            Console.WriteLine("Hello person two, let's compare you with person one.");
+
+            Person();
+
+
+            // End.
+            Console.WriteLine("\n");
+
+            Console.WriteLine("Judging by our calculations, we have come to the conclusion that both users are not within the healthy range " +
+                              "\nof the self-confidence crusher BMI Calculator.");
+
+            Console.WriteLine("\nHave a nice day!");
+            Console.ReadLine();
+        }
+
+
+        public static void Person()
```

Comment:
Please rename the function to be a verb and not a noun. 
HandlePerson, ProcessPerson sounds better ;)

Reply:
In addition:
We will make use of Person() (or some other nouns) later on. We don't want to mix functionnames with what comes then! :)
That is why this is important.